language-extensions/dotnet-core-CSharp/build/windows/create-dotnet-core-CSharp-extension-zip.cmd

```diff
@@ -24,18 +24,18 @@ REM Delete the ref folder so that the zip can be loaded by the SPEES
 RD /S /Q %BUILD_OUTPUT%\ref
 POPD
 
+REM Define files to compress, conditionally including .pdb files if BUILD_CONFIGURATION is "debug"
+SET FILES_TO_COMPRESS=%BUILD_OUTPUT%\*.dll, %BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.runtimeconfig.json, %BUILD_OUTPUT%\Microsoft.SqlServer.CSharpExtension.deps.json
```

Comment:
nit: leave an additional empty comment line after the comment to be consistent with the other comments within the file.
i.e. 
`REM`